core/tests/webdriverio_utils/BlogDashboardPage.js

```diff
@@ -108,6 +89,7 @@ var BlogDashboardPage = function() {
     await action.click(
       'Navigate back to blog dashboard button', navigateToBlogDashboardButton);
     await waitFor.pageToFullyLoad();
+    await waitFor.urlRedirection('http://localhost:9001/blog-dashboard');
```

Comment:
I think we don't need this because I just saw that a loading message appears when this this redirection is being done. 
![blog-load](https://user-images.githubusercontent.com/69013647/183017452-4ee36129-7af1-454c-9595-c6de5580eab1.JPG)

Reply:
Done.